src/cpp/src/sampler.hpp

```diff
@@ -294,17 +304,29 @@ SamplerOutput Sampler::sample(std::vector<SequenceGroup::Ptr> & sequence_groups,
                     running_sequence->append_token(sampled_token_id.m_index, sampled_token_id.m_log_prob);
                 };
                 for (size_t running_sequence_id = 0; running_sequence_id < num_running_sequences; ++running_sequence_id) {
+                    static ManualTimer timer1("sample::_get_logit_vector");
```

Comment:
Debug code?

Reply:
Yes, I added it to see what amount of time different operations take. 
Shall I remove all new timers from this PR?